README.md

```diff
@@ -1,6 +1,8 @@
 Dotbot
 ======
 
+[![Build Status](https://travis-ci.org/anishathalye/dotbot.svg?branch=master)](https://travis-ci.org/anishathalye/dotbot)
```

Comment:
Could you move this down a little bit (perhaps after the `---` separator)? It's not the most important thing for new users to see (ideally, the build should always be passing :stuck_out_tongue:)